config/foreman-proxy-content.migrations/200616155948-disable-pulp-3-proxying.rb

```diff
@@ -0,0 +1,15 @@
+FOREMAN_PROXY_CONTENT = 'foreman_proxy_content'.freeze
+PROXY_YUM = 'proxy_pulp_yum_to_pulpcore'.freeze
+PROXY_ISOS = 'proxy_pulp_isos_to_pulpcore'.freeze
+
+def upgrade
+  answers[FOREMAN_PROXY_CONTENT] = {} unless answers[FOREMAN_PROXY_CONTENT].is_a?(Hash)
+  answers[FOREMAN_PROXY_CONTENT][PROXY_YUM] = false
+  answers[FOREMAN_PROXY_CONTENT][PROXY_ISOS] = false
+end
+
+if answers[FOREMAN_PROXY_CONTENT] &&
+   answers[FOREMAN_PROXY_CONTENT][PROXY_YUM].nil? &&
+   answers[FOREMAN_PROXY_CONTENT][PROXY_ISOS].nil?
```

Comment:
```suggestion
if answers[FOREMAN_PROXY_CONTENT] == true ||
   (answers[FOREMAN_PROXY_CONTENT].is_a?(Hash) &&
   answers[FOREMAN_PROXY_CONTENT][PROXY_YUM].nil? &&
   answers[FOREMAN_PROXY_CONTENT][PROXY_ISOS].nil?)
```
I think this works in all cases.